pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -3488,6 +3488,17 @@ final: prev:
     meta.homepage = "https://github.com/travitch/hasksyn/";
   };
 
+  himalaya-vim = buildVimPluginFrom2Nix {
+    pname = "himalaya-vim";
+    version = "2022-10-10";
+    src = fetchgit {
+            url = "https://git.sr.ht/~soywod/himalaya-vim";
+            rev = "747ff2a6055839cb5b403b540883515af37b7117";
+            sha256 = "sha256-0WP9JoKN/A5bmnRjQJnquk3mRC7NEflUAdqki7+zqBU=";
+        };
```

Comment:
```suggestion
    src = fetchgit {
      url = "https://git.sr.ht/~soywod/himalaya-vim";
      rev = "747ff2a6055839cb5b403b540883515af37b7117";
      sha256 = "sha256-0WP9JoKN/A5bmnRjQJnquk3mRC7NEflUAdqki7+zqBU=";
    };
```

Reply:
@SuperSandro2000  This is a "bug" in the indentation of the Python script. You can check out other Vim plugins from SourceHut.

Reply:
https://github.com/NixOS/nixpkgs/blob/f0adf4fcae2aecdd6ade53cb5d4b94d3b7446efd/pkgs/applications/editors/vim/plugins/generated.nix#L3433-L3442

Here's an example of what I mean. Fixing this indentation bug is out of scope.